scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +656,98 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+
+class Slerp(object):
+    """Spherical Linear Interpolation of Rotations.
+
+    The SLERP algorithm guarantees that the interpolated rotations will have
+    constanct angular velocity between each successive "keyframe" rotation.
+
+    Given a set of `rotations` and corresponding fixed `timestamps`, this class
+    will initialize a `Slerp` instance which is an interpolator. The
+    interpolator can be called with another set of timestamps to return a
+    single `Rotation` object containing the interpolated rotations.
+
+    Parameters
+    ----------
+    rotations : a `Rotation` instance
```

Comment:
Add that `rotations` should contains (N,) rotations as well and at least two of them.